File: MPI/mpicheck/tests/functions/2.0/MPI_Status_set_cancelled.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Status *var_0;
    int var_1;
    int ret;
    /* calls */
    ret = MPI_Status_set_cancelled(var_0, var_1);
    ret = PMPI_Status_set_cancelled(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Status_set_cancelled.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0(10)
       LOGICAL var_1
       INTEGER var_2
        call mpi_status_set_cancelled(var_0, var_1, var_2)
       call pmpi_status_set_cancelled(var_0, var_1, var_2)
        end program main
    